presets/mainnet/electra.yaml

```diff
@@ -1,15 +1,36 @@
 # Mainnet preset - Electra
 
+# Gwei values
+# ---------------------------------------------------------------
+# 2**5 * 10**9 (= 32,000,000,000) Gwei
+MIN_ACTIVATION_BALANCE: 32000000000
+# 2**11 * 10**9 (= 2,048,000,000,000) Gwei
+MAX_EFFECTIVE_BALANCE_ELECTRA: 2048000000000
+
+# State list lengths
+# ---------------------------------------------------------------
+PENDING_BALANCE_DEPOSITS_LIMIT: 134217728
+PENDING_PARTIAL_WITHDRAWALS_LIMIT: 134217728
+PENDING_CONSOLIDATIONS_LIMIT: 262144
+
+# Reward and penalty quotients
+# ---------------------------------------------------------------
+MIN_SLASHING_PENALTY_QUOTIENT_ELECTRA: 4096
+WHISTLEBLOWER_REWARD_QUOTIENT_ELECTRA: 4096
+
 # # Max operations per block
 # ---------------------------------------------------------------
 # `uint64(2**0)` (= 1)
 MAX_ATTESTER_SLASHINGS_ELECTRA: 1
 # `uint64(2 * 3)` (= 8)
 MAX_ATTESTATIONS_ELECTRA: 8
+MAX_CONSOLIDATIONS: 1
```

Comment:
This includes a fix for `MAX_ATTESTATIONS_ELECTRA`.
```suggestion
# `uint64(2**3)` (= 8)
MAX_ATTESTATIONS_ELECTRA: 8
# `uint64(2**0)` (= 1)
MAX_CONSOLIDATIONS: 1
```

Reply:
I'm aware that this has been standard practice for now, but what novel information do these comments give? The type of the config value is defined in the specs